### PR TITLE
Add test case for mbb severity and fix error

### PR DESCRIPTION
**Pull request by thomascobb**
Also, when using `alarm.MINOR_ALARM` instead of `"MINOR"` I got:
```
Traceback (most recent call last):
  File "/dls/science/users/tmc43/common/python/pythonIoc/tests/sim_records.py", line 29, in <module>
    t_mbbi = mbbIn('MBBI', 'One', 'Two', ('Three', alarm.MINOR_ALARM), initial_value=2)
  File "/dls/science/users/tmc43/common/python/pythonIoc/softioc/builder.py", line 86, in mbbIn
    return _in_record('mbbi', name, **fields)
  File "/dls/science/users/tmc43/common/python/pythonIoc/softioc/builder.py", line 28, in _in_record
    return getattr(PythonDevice, record)(name, **fields)
  File "/dls/science/users/tmc43/common/python/pythonIoc/softioc/pythonSoftIoc.py", line 105, in __call__
    DTYP = self.dtyp, **fields)
  File "/dls/science/users/tmc43/common/python/pythonIoc/softioc/pythonSoftIoc.py", line 33, in __init__
    record = builder(name, **fields)
  File "/scratch/tmc43/pipenv/pythonIoc-y_I5oBLg/lib/python3.7/site-packages/epicsdbbuilder/recordbase.py", line 129, in __init__
    setattr(self, name, value)
  File "/scratch/tmc43/pipenv/pythonIoc-y_I5oBLg/lib/python3.7/site-packages/epicsdbbuilder/recordbase.py", line 195, in __setattr__
    self.__ValidateField(fieldname, value)
  File "/scratch/tmc43/pipenv/pythonIoc-y_I5oBLg/lib/python3.7/site-packages/epicsdbbuilder/recordbase.py", line 206, in __ValidateField
    self._validate.ValidFieldValue(fieldname, str(value))
  File "/scratch/tmc43/pipenv/pythonIoc-y_I5oBLg/lib/python3.7/site-packages/epicsdbbuilder/dbd.py", line 96, in ValidFieldValue
    'Can\'t write "%s" to field %s: %s' % (value, name, message)
AssertionError: Can't write "1" to field TWSV: Not a valid menu choice
```

### CODE REVIEW

**Comment by Araneidae** [comment]
Regarding using `alarm.MINOR_ALARM`, yes, I'm afraid that's how it works.  I think that putting in special hacks to work around this would be a terrible idea!  At least we get a pretty straightforward error message...

File: softioc/builder.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def process_value(prefix, value, option, severity=None):
         if severity:
             fields[prefix + 'SV'] = severity
     for prefix, (value, option) in zip(_mbbPrefixes, enumerate(options)):
-        if isinstance(value, tuple):
+        if isinstance(option, tuple):
```
Comment (Araneidae) [inline softioc/builder.py:75]: Ooooops.  Sorry, I clearly didn't test that.

**Comment by thomascobb** [comment]
> Regarding using `alarm.MINOR_ALARM`, yes, I'm afraid that's how it works. I think that putting in special hacks to work around this would be a terrible idea! At least we get a pretty straightforward error message...

I was wondering if this was new behaviour, as the original docs and your updated docs both mentioned using `alarm.MIINOR_ALARM` when specifying mbbi/o records...

**Comment by thomascobb** [comment]
@Araneidae can I merge this one and make a 3.0 release? I think everything else is sorted now

**Comment by Araneidae** [comment]
Go ahead!

File: docs/reference/api.rst [hunk 2]
```diff
@@ -273,8 +273,8 @@ All functions return a wrapped `ProcessDeviceSupportIn` or
 
         status = mbbIn('STATUS',
             'OK',
-            ('FAILING', alarm.MINOR_ALARM),
-            ('FAILED', alarm.MAJOR_ALARM))
```
Comment (Araneidae) [inline docs/reference/api.rst:277]: Was this in the original documentation?  Was a bug if so.

Comment (thomascobb) [reply docs/reference/api.rst:277]: Yes

**Pull request merged**